labgrid/driver/dediprogdriver.py

```diff
@@ -0,0 +1,67 @@
+import os.path
+import logging
+import attr
+
+from ..resource import NetworkDediprog
+from ..factory import target_factory
+from ..step import step
+from ..protocol import BootstrapProtocol
+from .common import Driver, check_file
+from ..util.managedfile import ManagedFile
+from ..util.helper import processwrapper
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class DediprogDriver(Driver, BootstrapProtocol):
+    """ The Dediprog driver used the dediprog utility to write an image to a raw rom.
+    The driver is a pure wrapper of the dpcmd utility"""
+    bindings = {
+        'dediprog_resource': {"Dediprog", NetworkDediprog},
+    }
+
+    image = attr.ib(default=None)
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        self.logger = logging.getLogger(f'{self}')
+        if self.target.env:
+            self.tool = self.target.env.config.get_tool('dpcmd')
+        else:
+            self.tool = 'dpcmd'
+        self.logger.debug('Tool %s', self.tool)
+
+    def _get_dediprog_prefix(self):
+        return self.dediprog_resource.command_prefix+[
+            self.tool
+        ]
+
+    def on_activate(self):
+        pass
+
+    def on_deactivate(self):
+        pass
+
+    @Driver.check_active
+    @step(title='call', args=['args'])
+    def __call__(self, *args):
+        arg_list = list(args)
+        arg_list.append('--vcc')
+        arg_list.append(f'{self.dediprog_resource.vcc}')
+        arg_list.append('--silent')
```

Comment:
Is there still enough output to debug problems even with `--silent`?

Reply:
Yep. The `--silent` just prevents a "fancy" time counter.